modules/misc/qt.nix

```diff
@@ -159,8 +159,10 @@ in {
 
     # Necessary because home.sessionVariables doesn't support mkIf
     envVars = lib.filterAttrs (n: v: v != null) {
-      QT_QPA_PLATFORMTHEME =
-        styleNames.${cfg.platformTheme} or cfg.platformTheme;
+      QT_QPA_PLATFORMTHEME = if (cfg.platformTheme != null) then
+        styleNames.${cfg.platformTheme} or cfg.platformTheme
```

Comment:
This doesn't really work. Try setting `platformTheme = "gtk3"`, it will complain about `gtk3` attribute missing from `styleNames`.

Reply:
```
       error: attribute 'gtk3' missing

       at /nix/store/91m8mdr4lxwjm3h3gz6xribrd0pi7rvp-source/modules/misc/qt.nix:212:7:

          211|     home.packages = (lib.optionals (cfg.platformTheme != null)
          212|       platformPackages.${cfg.platformTheme})
             |       ^
          213|       ++ (lib.optionals (cfg.style.package != null)
       Did you mean gtk?
```

Probably nix just trying to be too smart 😅.

Reply:
Good catch, I forgot to add the fallback here.

Will also add another test, otherwise this get too easy to miss in future changes.